### PR TITLE
pajkossy_1996_impedance_209

### PR DESCRIPTION
**Pull request by markus-chem**
add lit folder

### CODE REVIEW

File: literature/pajkossy_1996_impedance_209/pajkossy_1996_impedance_209_p2_1A_dotted.yaml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+            type: SCE # the reference electrode given in the experimental section. Might differ from that in the plot, MSE-sat, MSE-3M, MSE-1M, RHE, AgCl2
+            source:
+                supplier:
+        counter electrode:
+            material: Pt
+            crystallographic orientation:
+            shape: wire
+        working electrode:
+            material: Au
+            crystallographic orientation: 111 # hkl, 100, 110, poly
```
Comment (DunklesArchipel) [inline literature/pajkossy_1996_impedance_209/pajkossy_1996_impedance_209_p2_1A_dotted.yaml:87]:
```suggestion
            crystallographic orientation: '111' # hkl, 100, 110, poly

```

File: literature/pajkossy_1996_impedance_209/pajkossy_1996_impedance_209_p2_1A_solid.yaml [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+            type: SCE # the reference electrode given in the experimental section. Might differ from that in the plot, MSE-sat, MSE-3M, MSE-1M, RHE, AgCl2
+            source:
+                supplier:
+        counter electrode:
+            material: Pt
+            crystallographic orientation:
+            shape: wire
+        working electrode:
+            material: Au
+            crystallographic orientation: 111 # hkl, 100, 110, poly
```
Comment (DunklesArchipel) [inline literature/pajkossy_1996_impedance_209/pajkossy_1996_impedance_209_p2_1A_solid.yaml:87]:
```suggestion
            crystallographic orientation: '111' # hkl, 100, 110, poly

```

File: literature/pajkossy_1996_impedance_209/pajkossy_1996_impedance_209_p2_1B_dotted.yaml [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+            type: SCE # the reference electrode given in the experimental section. Might differ from that in the plot, MSE-sat, MSE-3M, MSE-1M, RHE, AgCl2
+            source:
+                supplier:
+        counter electrode:
+            material: Pt
+            crystallographic orientation:
+            shape: wire
+        working electrode:
+            material: Au
+            crystallographic orientation: 100 # hkl, 100, 110, poly
```
Comment (DunklesArchipel) [inline literature/pajkossy_1996_impedance_209/pajkossy_1996_impedance_209_p2_1B_dotted.yaml:87]:
```suggestion
            crystallographic orientation: '100' # hkl, 100, 110, poly

```

File: literature/pajkossy_1996_impedance_209/pajkossy_1996_impedance_209_p2_1B_solid.yaml [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+            type: SCE # the reference electrode given in the experimental section. Might differ from that in the plot, MSE-sat, MSE-3M, MSE-1M, RHE, AgCl2
+            source:
+                supplier:
+        counter electrode:
+            material: Pt
+            crystallographic orientation:
+            shape: wire
+        working electrode:
+            material: Au
+            crystallographic orientation: 100 # hkl, 100, 110, poly
```
Comment (DunklesArchipel) [inline literature/pajkossy_1996_impedance_209/pajkossy_1996_impedance_209_p2_1B_solid.yaml:87]:
```suggestion
            crystallographic orientation: '100' # hkl, 100, 110, poly

```

File: literature/pajkossy_1996_impedance_209/pajkossy_1996_impedance_209_p3_3A_a.yaml [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+            type: SCE # the reference electrode given in the experimental section. Might differ from that in the plot, MSE-sat, MSE-3M, MSE-1M, RHE, AgCl2
+            source:
+                supplier:
+        counter electrode:
+            material: Pt
+            crystallographic orientation:
+            shape: wire
+        working electrode:
+            material: Au
+            crystallographic orientation: 111 # hkl, 100, 110, poly
```
Comment (DunklesArchipel) [inline literature/pajkossy_1996_impedance_209/pajkossy_1996_impedance_209_p3_3A_a.yaml:87]:
```suggestion
            crystallographic orientation: '111' # hkl, 100, 110, poly

```

File: literature/pajkossy_1996_impedance_209/pajkossy_1996_impedance_209_p3_3A_b.yaml [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+            type: SCE # the reference electrode given in the experimental section. Might differ from that in the plot, MSE-sat, MSE-3M, MSE-1M, RHE, AgCl2
+            source:
+                supplier:
+        counter electrode:
+            material: Pt
+            crystallographic orientation:
+            shape: wire
+        working electrode:
+            material: Au
+            crystallographic orientation: 111 # hkl, 100, 110, poly
```
Comment (DunklesArchipel) [inline literature/pajkossy_1996_impedance_209/pajkossy_1996_impedance_209_p3_3A_b.yaml:87]:
```suggestion
            crystallographic orientation: '111' # hkl, 100, 110, poly

```

File: literature/pajkossy_1996_impedance_209/pajkossy_1996_impedance_209_p3_3A_dashed.yaml [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+            type: SCE # the reference electrode given in the experimental section. Might differ from that in the plot, MSE-sat, MSE-3M, MSE-1M, RHE, AgCl2
+            source:
+                supplier:
+        counter electrode:
+            material: Pt
+            crystallographic orientation:
+            shape: wire
+        working electrode:
+            material: Au
+            crystallographic orientation: 111 # hkl, 100, 110, poly
```
Comment (DunklesArchipel) [inline literature/pajkossy_1996_impedance_209/pajkossy_1996_impedance_209_p3_3A_dashed.yaml:87]:
```suggestion
            crystallographic orientation: '111' # hkl, 100, 110, poly

```

File: literature/pajkossy_1996_impedance_209/pajkossy_1996_impedance_209_p6_8A_a.yaml [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+            type: SCE # the reference electrode given in the experimental section. Might differ from that in the plot, MSE-sat, MSE-3M, MSE-1M, RHE, AgCl2
+            source:
+                supplier:
+        counter electrode:
+            material: Pt
+            crystallographic orientation:
+            shape: wire
+        working electrode:
+            material: Au
+            crystallographic orientation: 100 # hkl, 100, 110, poly
```
Comment (DunklesArchipel) [inline literature/pajkossy_1996_impedance_209/pajkossy_1996_impedance_209_p6_8A_a.yaml:87]:
```suggestion
            crystallographic orientation: '100' # hkl, 100, 110, poly

```

File: literature/pajkossy_1996_impedance_209/pajkossy_1996_impedance_209_p6_8A_b.yaml [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+            type: SCE # the reference electrode given in the experimental section. Might differ from that in the plot, MSE-sat, MSE-3M, MSE-1M, RHE, AgCl2
+            source:
+                supplier:
+        counter electrode:
+            material: Pt
+            crystallographic orientation:
+            shape: wire
+        working electrode:
+            material: Au
+            crystallographic orientation: 100 # hkl, 100, 110, poly
```
Comment (DunklesArchipel) [inline literature/pajkossy_1996_impedance_209/pajkossy_1996_impedance_209_p6_8A_b.yaml:87]:
```suggestion
            crystallographic orientation: '100' # hkl, 100, 110, poly

```

File: literature/pajkossy_1996_impedance_209/pajkossy_1996_impedance_209_p6_8A_dashed.yaml [hunk 10]
```diff
@@ -0,0 +1,102 @@
...
+            type: SCE # the reference electrode given in the experimental section. Might differ from that in the plot, MSE-sat, MSE-3M, MSE-1M, RHE, AgCl2
+            source:
+                supplier:
+        counter electrode:
+            material: Pt
+            crystallographic orientation:
+            shape: wire
+        working electrode:
+            material: Au
+            crystallographic orientation: 100 # hkl, 100, 110, poly
```
Comment (DunklesArchipel) [inline literature/pajkossy_1996_impedance_209/pajkossy_1996_impedance_209_p6_8A_dashed.yaml:87]:
```suggestion
            crystallographic orientation: '100' # hkl, 100, 110, poly

```

**Review by DunklesArchipel — CHANGES_REQUESTED** [review]
The crystallographic orientation was missing some ' '.
I Hopefully added all of them. Please check carefully.

**Comment by markus-chem** [comment]
All hkl changed to string.

**Comment by markus-chem** [comment]
> The crystallographic orientation was missing some ' '. I Hopefully added all of them. Please check carefully.

Done and checked.

**Review by DunklesArchipel — APPROVED** [review]
seems fine

**Pull request merged**